src/main/java/au/com/dmg/fusion/request/paymentrequest/PaymentRequest.java

```diff
@@ -86,6 +96,25 @@ public PaymentRequest.Builder paymentData(PaymentData paymentData) {
             return PaymentRequest.Builder.this;
         }
 
+        public Builder customFields(List<CustomField> customFields) {
+            if(customFields == null){
+                this.customFields = new ArrayList<>();
+            } else {
+                this.customFields = customFields;
+            }
+            return PaymentRequest.Builder.this;
+        }
+
+        public PaymentRequest.Builder addCustomField(CustomField customField) {
+            this.customFields.add(customField);
+            return PaymentRequest.Builder.this;
+        }
+
+        public PaymentRequest.Builder addCustomFields(List<CustomField> customFields) {
+            this.customFields.addAll(customFields);
```

Comment:
Please check if customFields (parameter value) is null before adding to this.customFields.